### ISSUE TITLE
Adjustment to the Server Reset Time

### ISSUE THREAD

**Issue by Khyrberos** [magefree/mage#5375]
TL;DR - I would inquire as to the feasibility of moving the server reset time (currently 8am; 2am EST) back an hour (or even two; 9/10am (3/4am EST)).  (This presupposes that anyone here is in control of that setting, and that it is even possible to change at this point, of course.)

I am very aware of the self-aggrandizing nature of this request, and would not otherwise ask it, except that I figured 'what can it hurt'.
Also, I figured that while it was possible the current server reset time was chosen specifically, it's also very possible that it was chosen _arbitrarily_, i.e "some time in the early morning when the least number of locals (German/Europeans I presume) are on".
If the latter is the case, then it stands to reason it wouldn't matter _too_ much to whoever's-in-charge to back it up an hour or two.

For me personally (being in the US, Eastern time-zone), it would be a big deal & a great favor; 2am is kinda right in the middle of things for me, but 3/4a is sufficiently late that I should be going to bed anyway, and would just really help me out generally.  And really for anybody in the U.S., since the current server reset time is at 11pm for the Western coast (PST), which is even more prohibitive for many, I imagine.

I appreciate your consideration of my request.

**Comment by Khyrberos** [comment]
Hm. I figured that was the case.  However, is 8am your time the absolute lowest traffic-time, or is it a range/does it vary day to day?

**Comment by Zerrisx** [comment]
This doesn't seem to need any action taken on it.
